### PR TITLE
Updated to User Guide formatting

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the formatting for almost every topic in the Sales-Channels > ASC folder. Staging build #385

## Affected documentation pages

Amazon Sales Channel 4.0+ section of the Marketing menu.

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

### CODE REVIEW

File: src/sales-channels/asc/business-pricing.md [hunk 1]
```diff
@@ -39,6 +39,6 @@ _Business Pricing (B2B)_
...
+|Enable Business Pricing|Options: <br/>**Disabled** - Choose when you do not want to enable business to business sales. All other fields in this section are disabled when chosen.<br/>**Enabled** - Choose when you want to enable your business to business sales. When enabled, the business price is set equal to the list price after all pricing rules have been applied. The business price follows the website pricing scope, if enabled. A business price can not be less than $1. |
+|Enable Tiered Pricing|(Required) Options: <br/>**Disabled** - Choose when you want the same listing price for all order quantities. All Pricing Level fields in this section are disabled when chosen.<br/>**Enabled** - Choose when you want to enable pricing that adjusts based on order quantity. When enabled, the Pricing Level quantity and discount fields enable. |
```
Comment (leslietilling) [inline src/sales-channels/asc/business-pricing.md]:
```suggestion
|Enable Tiered Pricing|(Required) Options: <br/>**Disabled** - Choose when you want the same listing price for all order quantities. When chosen, all Pricing Level fields in this section are disabled.<br/>**Enabled** - Choose when you want to enable pricing that adjusts based on order quantity. When chosen, the Pricing Level quantity and discount fields are enabled. |
```

File: src/sales-channels/asc/common-order-processing.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
+Amazon Sales Channel can manage your Amazon orders, including emailing the buyer, fulfilling the order (shipping), issuing credits/refunds, adding comments, and more. To manage your Amazon orders, your [order settings]({% link sales-channels/asc/order-settings.md %}) must be set to create corresponding Magento orders when Amazon orders are received. Amazon order information shows on the _Orders_ tab on the Amazon Sales Channel home page.
 
-When corresponding Magento orders are created for Amazon orders, the assigned number displays in the Magento Order Number column. Clicking the order number opens the order in the Magento [order processing]({% link sales/order-processing.md %}) screen in a new tab. You can process the order as you do your other Magento orders.
+When corresponding Magento orders are created for Amazon orders, the assigned number shows in the _Magento Order Number_ column. Click the order number tp open the order in the Magento [order processing]({% link sales/order-processing.md %}) page in a new tab. You can process the order as you do your other Magento orders.
```
Comment (leslietilling) [inline src/sales-channels/asc/common-order-processing.md]:
```suggestion
When corresponding Magento orders are created for Amazon orders, the assigned number shows in the _Magento Order Number_ column. Click the order number to open the order in the Magento [order processing]({% link sales/order-processing.md %}) page in a new tab. You can process the order as you do your other Magento orders.
```

File: src/sales-channels/asc/end-listings-manually.md [hunk 3]
```diff
@@ -3,22 +3,22 @@ title: Ending an Amazon Listing
...
 
-1. In the Action column for the listing, click **Select** and then click **End Listing**.
+1. Under _Action_, click **Select** > **End Listing**.
 
 1. Click **OK** on the confirmation message.
 
-When ending a single listing or multiple listings, a confirmation message displays that the selected listing is being processed to end. Ended listings can be managed on the [Ended]({% link sales-channels/asc/ended-listings.md %}) tab in Manage Listings. Once processed and removed from Amazon, the status for the listing will display as "Manually Ended." Ended listings will remain removed from Amazon until you manually republish the listing using the Republish to Amazon action.
+When ending a single listing or multiple listings, a message appears confirming that the selected listing is being processed to end. Ended listings can be managed on the [Ended]({% link sales-channels/asc/ended-listings.md %}) tab in Manage Listings. Once processed and removed from Amazon, the status for the listing will display as "Manually Ended." Ended listings will remain removed from Amazon until you manually republish the listing using the Republish to Amazon action.
```
Comment (leslietilling) [inline src/sales-channels/asc/end-listings-manually.md]:
```suggestion
When ending a single listing or multiple listings, a message confirms that the selected listing is being processed to end. Ended listings can be managed on the [Ended]({% link sales-channels/asc/ended-listings.md %}) tab in Manage Listings. When processed and removed from Amazon, the status for the listing will appear as "Manually Ended." Ended listings will remain removed from Amazon until you manually republish the listing using the Republish to Amazon action.
```

File: src/sales-channels/asc/ended-listings.md [hunk 4]
```diff
@@ -3,45 +3,45 @@ title: Ended Listings
...
 
-1. In the Action column for the listing, click **Select** and click **End Listing**
+1. In the Action column for the listing, click **Select** > **End Listing**
 
-1. Click **OK** on the confirmation message that displays.
+1. Click **OK** on the confirmation message.
 
-When ending a single listing or multiple listings, a confirmation message displays that the selected listing is being processed to end. Ended listings can be managed on Ended tab in Manage Listings. Once processed and removed from Amazon, the status for the listing will display as "Manually Ended". Ended listings will remain removed from Amazon until you manually republish the listing using the Republish to Amazon action.
+When ending a single listing or multiple listings, a message appears confirming that the selected listing is being processed to end. Ended listings can be managed on Ended tab in Manage Listings. Once processed and removed from Amazon, the status for the listing will display as "Manually Ended". Ended listings will remain removed from Amazon until you manually republish the listing using the Republish to Amazon action.
```
Comment (leslietilling) [inline src/sales-channels/asc/ended-listings.md]:
```suggestion
When ending a single listing or multiple listings, a message confirms that the selected listing is being processed to end. Ended listings can be managed on Ended tab in Manage Listings. When processed and removed from Amazon, the status for the listing will appear as "Manually Ended". Ended listings will remain removed from Amazon until you manually republish the listing using the Republish to Amazon action.
```

File: src/sales-channels/asc/listing-price.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ The options listed in Magento Price Source, Minimum Advertised Price (Map), and
 
 1. For **Currency Conversion**, choose an option. The default is `Disabled`.
 
-   The options in this drop-down are dependent on your [Magento currency]({% link configuration/general/currency-setup.md %}) settings. If no options are available, you may need to set up your currency settings.
+   The options in this drop-down depend on your [Magento currency]({% link configuration/general/currency-setup.md %}) settings. If no options are available, you may need to set up your currency settings.
```
Comment (leslietilling) [inline src/sales-channels/asc/listing-price.md]:
```suggestion
   The available options depend on your [Magento currency]({% link configuration/general/currency-setup.md %}) settings. If no options are available, you may need to set up your currency settings.
```

File: src/sales-channels/asc/listing-rules.md [hunk 6]
```diff
@@ -38,7 +38,7 @@ _Listing Rules_
 
 |Field|Description|
 |---|---|
-|Websites|The options in this drop-down are dependent on the [websites]({% link stores/websites-stores-views.md %}) you have set up in your Magento configuration. Select the website to represent which website to have eligible products listed on Amazon. Only one website can be selected, as each website requires a unique Amazon store created in Amazon Sales Channel. |
+|Websites|The options in this drop-down depend on the [websites]({% link stores/websites-stores-views.md %}) you have set up in your Magento configuration. Select the website to represent which website to have eligible products listed on Amazon. Only one website can be selected, as each website requires a unique Amazon store created in Amazon Sales Channel. |
```
Comment (leslietilling) [inline src/sales-channels/asc/listing-rules.md]:
```suggestion
|Websites|The available options depend on the [websites]({% link stores/websites-stores-views.md %}) you have set up in your Magento configuration. Select the website to represent which website to have eligible products listed on Amazon. Only one website can be selected, as each website requires a unique Amazon store created in Amazon Sales Channel. |
```

File: src/sales-channels/asc/managing-listings-by-tab.md [hunk 7]
```diff
@@ -12,20 +12,19 @@ title: Managing Product Listings by Tab
...
+|[Ended]({% link sales-channels/asc/ended-listings.md %})|Shows your Amazon listings that have manually been ended (removed) from Amazon.|[Publish Product to Amazon]({% link sales-channels/asc/ended-listings.md %})<br/>[View Details]({% link sales-channels/asc/product-listing-details.md %})<br/>[Publish On Amazon]({% link sales-channels/asc/ended-listings.md %})<br/>[Create Alias Seller SKU]({% link sales-channels/asc/create-alias-seller-sku.md %})|
 
+## Access Product Listings
 
-## To access Product Listings:
+1. On the _Admin_ sidebar, click **Marketing** > _Channels_ > **Amazon**.
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-listings-by-tab.md]:
```suggestion
1. On the _Admin_ sidebar, go to **Marketing** > _Channels_ > **Amazon**.
```

File: src/sales-channels/asc/managing-orders.md [hunk 8]
```diff
@@ -16,23 +16,25 @@ Orders that are not imported:
...
 
-When managing Amazon orders in Amazon Sales Channel, you will use the [Magento Order Management]({% link sales/order-management.md %}) processes and workflows. When you click the Magento order number for the order, the order displays in a new tab the same as if you accessed the order in Sales grid in the admin panel.
+When managing Amazon orders in Amazon Sales Channel, you will use the [Magento Order Management]({% link sales/order-management.md %}) processes and workflows. When you click the Magento order number for the order, the order appears in a new tab the same as if you accessed the order in Sales grid in the admin panel.
 
-## To view your Amazon order information:
+## View your Amazon order information
 
-1. On the Admin sidebar, click **Marketing**. Then, under **Channels**, click **Amazon**.
+1. On the _Admin_ sidebar, click **Marketing** > _Channels_ > **Amazon**.
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-orders.md]:
```suggestion
1. On the _Admin_ sidebar, go to **Marketing** > _Channels_ > **Amazon**.
```

File: src/sales-channels/asc/managing-product-listings.md [hunk 9]
```diff
@@ -3,18 +3,18 @@ title: Managing Amazon Listings
...
 
 [Manage Listings by Status / Tab]({% link sales-channels/asc/managing-listings-by-tab.md %})
 
 [Manage Listings by Action]({% link sales-channels/asc/managing-listings-by-action.md %})
 
-## To access Product Listings:
+## Access product listings
 
-1. On the Admin sidebar, click **Marketing**. Then, under **Channels**, click **Amazon**.
+1. On the _Admin_ sidebar, click **Marketing** > _Channels_ > **Amazon**.
```
Comment (leslietilling) [inline src/sales-channels/asc/managing-product-listings.md]:
```suggestion
1. On the _Admin_ sidebar, go to **Marketing** > _Channels_ > **Amazon**.
```

File: src/sales-channels/asc/ob-store-review.md [hunk 10]
```diff
@@ -6,7 +6,7 @@ After you connect your Amazon Sales Channel store, you should review your store'
 
 ## Advanced Store Settings
 
-1. On the Admin sidebar, click **Marketing** > _Channels_ > **Amazon**.
+1. On the _Admin_ sidebar, click **Marketing** > _Channels_ > **Amazon**.
```
Comment (leslietilling) [inline src/sales-channels/asc/ob-store-review.md]:
```suggestion
1. On the _Admin_ sidebar, go to **Marketing** > _Channels_ > **Amazon**.
```

File: src/sales-channels/asc/workspace-controls.md [hunk 11]
```diff
@@ -10,31 +10,31 @@ _Amazon Sales Channel Workspace Controls Examples_
...
+- Orders with a buyer named of `Smith`, and
+- Orders with a status of `Shipped`.
 
 When you have your filtering options set, click **Apply Filters** to filter the data listed. Click Cancel to exit the Filters control without applying.
 
 ![]({% link sales-channels/asc/assets/workspace-controls-filters.png %}){: .zoom}
 _Filters Control Example_
 
-Once you've applied filters to your data, **Active Filters** information will display. You can click the ![]({% link sales-channels/asc/assets/x-icon-clear-filters.png %}){: .Inline}icon to clear a specific filter option or click **Clear All** to clear all applied filters.
+Once you've applied filters to your data, **Active Filters** information will appear. You can click the ![]({% link sales-channels/asc/assets/x-icon-clear-filters.png %}){: .Inline}icon to clear a specific filter option or click **Clear All** to clear all applied filters.
```
Comment (leslietilling) [inline src/sales-channels/asc/workspace-controls.md]:
```suggestion
After you apply filters to your data, **Active Filters** information will appear. You can click the ![]({% link sales-channels/asc/assets/x-icon-clear-filters.png %}){: .Inline}icon to clear a specific filter option or click **Clear All** to clear all applied filters.
```

**Comment by hickskenh** [comment]
running tests

**Pull request merged**